### ISSUE TITLE
toggle switch usability/accessibility tweaks

### ISSUE THREAD

**Issue by Blackbaud-ToddRoberts** [blackbaud/skyux-forms#56]
### Expected behavior
When mousing over disabled toggles, the cursor uses the "not-allowed" style
Disabled toggles cannot be focused

### Actual behavior
Disabled toggles have normal cursor on hover
Disabled toggles are focusable

**Comment by Blackbaud-SteveBrush** [comment]
Talked with Todd; we'll adjust the cursor for "readonly" states, but still allow the toggle to be focusable.
